volume/local/local.go

```diff
@@ -58,14 +60,18 @@ type Root struct {
 	volumes map[string]*Volume
 }
 
+// DataPath returns the constructed path of this volume.
 func (r *Root) DataPath(volumeName string) string {
 	return filepath.Join(r.path, volumeName, VolumeDataPathName)
 }
 
+// Name returns name of this (driver?)
 func (r *Root) Name() string {
-	return "local"
+	return volume.DefaultDriverName
 }
 
+// Create makes a local directory to associate with the returned
+// volume. Returns an error if the volume already exists.
```

Comment:
Create creates a new `volume.Volume` with the provided name, creating the underlying directory tree required for this volume in the process.